### DISCUSSION TITLE
❗ Looking for new maintainers

### DISCUSSION THREAD

**Discussion by joschi** [OpenAPITools/openapi-diff#598] [Announcements]
The openapi-diff project currently has 700 stars and numerous users rely on its functionality.

Unfortunately I have not been using openapi-diff myself for some time and and I don't have time to properly maintain and develop this project anymore, so I'm looking for one or more people to take over maintenance of openapi-diff.

If you're interested in joining the project, please reach out (via this discussion thread).

Cheers,
joschi

**Comment by dsinghvi** [comment]
@joschi i'm interested in maintaining

**Comment by stoeren** [comment]
I am also interested in maintaining this repo

**Comment by arcesino** [comment]
I'm interested in becoming a maintainer of this repo

**Comment by SachinAkash01** [comment]
@joschi I'm also interested in maintaining this repo..

**Comment by DrSatyr** [comment]
@joschi any updates on this topic? There are a lot of people who wish to maintain the project. My project depends on this library so I am interested in active maintenance.

**Comment by joschi** [comment]
@dsinghvi @stoeren @arcesino @SachinAkash01 @DrSatyr Thank you so much for showing interest in contributing to this project! ❤️ 

There are a bunch of [open issues](https://github.com/OpenAPITools/openapi-diff/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) and [pull requests](https://github.com/OpenAPITools/openapi-diff/pulls) which could need some love and reviews. Please check if you can contribute to any of these.

Other than that, please join the OpenAPI Tools Slack via https://openapi-generator.tech/#connectOnSlack to connect to the rest of the team and potentially set up the maintainer permissions on this repository

**Comment by wing328** [comment]
We can grant the merge right (collaborator) to those who've 3 merged PRs to start with.

If you've at least 3 merged PRs and want the merge right, please reply to let us know.

**Reply by DrSatyr** [reply]
Hello @wing328,

I’m currently using the apidiff library in my project as a dependency and have submitted three PRs to address issues that we’ve encountered:

- https://github.com/OpenAPITools/openapi-diff/pull/718
- https://github.com/OpenAPITools/openapi-diff/pull/671
- https://github.com/OpenAPITools/openapi-diff/pull/661

Unfortunately, none of them have been merged yet, which creates a bit of a catch-22, as I don’t meet the requirement for merge rights. If there’s no one else available to support this project, I’d be happy to help maintain it and try to ensure ongoing support and improvements.

Thank you for considering this!

**Reply by wing328** [reply]
@DrSatyr sorry that I missed your reply.

Can you please PM me via Slack and we will figure out a way to move this project forward by reviewing and merging your PRs?

https://join.slack.com/t/openapi-generator/shared_invite/zt-2wmkn4s8g-n19PJ99Y6Vei74WMUIehQA

**Comment by DrSatyr** [comment]
Hello everyone, I’m excited to share that I’ve recently been granted maintainer rights for this repository. Let’s keep moving forward – the show must go on!
